src/backend/distributed/relay/relay_event_utility.c

```diff
@@ -536,6 +547,70 @@ AppendShardIdToConstraintName(AlterTableCmd *command, uint64 shardId)
 }
 
 
+static bool
+ExtendRowReferencesWalker(Node *node, RowRefWalkerState *state)
+{
+	if (node == NULL)
```

Comment:
Can you declare a `bool` named `walkerResult` to change this walker to have a single return point? See for example `ExtractRangeTableIndexWalker`.